srcCxx/fast-dds-cmake/CMakeLists.txt

```diff
@@ -24,31 +22,40 @@ find_program(FASTDDSGEN fastddsgen)
 
 # Generate code from IDL
 file(MAKE_DIRECTORY ${CMAKE_SOURCE_DIR}/../GeneratedCode)
+set(
+  GENERATED_TYPE_SUPPORT_SOURCES
+  ${CMAKE_SOURCE_DIR}/../shape.cxx
+  ${CMAKE_SOURCE_DIR}/../shapePubSubTypes.cxx
+  ${CMAKE_SOURCE_DIR}/../shapeTypeObject.cxx
+)
 set(
   GENERATED_TYPE_SUPPORT_FILES
-  ${CMAKE_SOURCE_DIR}/../GeneratedCode/shape.h
-  ${CMAKE_SOURCE_DIR}/../GeneratedCode/shape.cxx
-  ${CMAKE_SOURCE_DIR}/../GeneratedCode/shapePubSubTypes.h
-  ${CMAKE_SOURCE_DIR}/../GeneratedCode/shapePubSubTypes.cxx
-  ${CMAKE_SOURCE_DIR}/../GeneratedCode/shapeTypeObject.h
-  ${CMAKE_SOURCE_DIR}/../GeneratedCode/shapeTypeObject.cxx
+  ${CMAKE_SOURCE_DIR}/../shape.cxx
+  ${CMAKE_SOURCE_DIR}/../shape.h
+  ${CMAKE_SOURCE_DIR}/../shapeCdrAux.hpp
+  ${CMAKE_SOURCE_DIR}/../shapeCdrAux.ipp
+  ${CMAKE_SOURCE_DIR}/../shapePubSubTypes.cxx
+  ${CMAKE_SOURCE_DIR}/../shapePubSubTypes.h
+  ${CMAKE_SOURCE_DIR}/../shapeTypeObject.cxx
+  ${CMAKE_SOURCE_DIR}/../shapeTypeObject.h

```

Comment:
probably you may want to add these files to the .gitignore

Reply:
It seems I pushed the wrong version of my `CMakeLists.txt`. 
Fixed in 3165556b174af44c3cf0387e8af06774ec1e4115

Reply:
The code is generated in `srcCxx/GeneratedCode`, which is already ignored